.github/workflows/manual_ghcr.yaml

```diff
@@ -0,0 +1,42 @@
+name: Publish docker image to ghcr
+on: workflow_dispatch
+jobs:
+  push_to_registry:
+    name: Push Docker image to GitHub Packages
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Get docker tag
+        run: echo "::set-output name=docker_tag::${GITHUB_REF##*/}_${{github.sha}}"
+        id: docker_tag
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to Container Registry
+        uses: docker/login-action@v1
+        with:
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.APPKNOX_GITHUB_CONTAINER_TOKEN }} #encrypted secret var
+          registry: ${{ github.event.inputs.registry }}
```

Comment:
`github.event.inputs.registry` this should not be manual input right?

Reply:
ya this is a mistake ghrc.io is hardcoded now